src/app/common/ErrorBlock.vue

```diff
@@ -106,12 +106,12 @@ const invalidParameters = computed(() => props.error instanceof ApiError ? props
 
 .badge-list {
   position: absolute;
-  top: var(--spacing-xs);
-  right: var(--spacing-xs);
+  top: $kui-space-40;
+  right: $kui-space-40;
```

Comment:
issue: you must not use spacing tokens for position attributes. 

You should enable the stylelint plugin to enforce the proper usage of tokens

Reply:
That’s actually a rule I’d like to challenge. In a lot of cases, positioning an element happens such that it aligns in relation with spacing properties like that.

Not being able to use space variables for this would just risk misalignment issues should those values every change.

Reply:
This was an architecture decision with design.